### ISSUE TITLE
Reset 2-factor authentication

### ISSUE THREAD

**Issue by ifathy** [MicrosoftDocs/feedback#2936]
hi, 

How do I reset my 2-factor authentication on Microsoft authenticator?

I use MS Teams and Mavim also for business needsand needs and I did setup 2-factor authentication using Microsoft authenticator on my old phone, now my old phone's software got reset due to irrelevant reasons, and I'm unable to login to organizations which require 2-factor authentication , and I have not been able to add my business account to authenticator again on my phone as it requires Microsoft website to show me a QR code to scan with my phone through authenticator which does not happen

So in short again, how do I reset my Microsoft 2-factor authentication.

Thanks, stay safe

**Comment by ifathy** [comment]
Added screenshot for clarification

**Comment by ifathy** [comment]
![Screenshot_20200710-105430_Authenticator](https://user-images.githubusercontent.com/29517649/87136688-22a2c200-c29c-11ea-9dc0-4898deb902ca.jpg)

**Comment by ifathy** [comment]
![Screenshot_20200710-105449_Authenticator](https://user-images.githubusercontent.com/29517649/87136715-2c2c2a00-c29c-11ea-8a71-e4afe5ab6bd7.jpg)
